python/nano/src/bigdl/nano/utils/inference/pytorch/model_utils.py

```diff
@@ -82,3 +84,37 @@ def export_to_onnx(model, input_sample=None, onnx_path="model.onnx", dynamic_axe
                                 }
     default_onnx_export_args.update(kwargs)
     torch.onnx.export(model, input_sample, onnx_path, **default_onnx_export_args)
+
+
+class FP32InferenceModelWrapped(pl.LightningModule):
+    def __init__(self, model, inference_method_name="forward"):
+        super().__init__()
+        self.model = model
+        self.inference_method_name = inference_method_name
+
+    def forward(self, *args):
+        method_to_call = getattr(self.model, self.inference_method_name)
```

Comment:
does it work if `self.inference_method_name="forward"`?

Reply:
yeah it works, still to avoid any new problems involved by this wrapper, we will not wrap users' model if users use "forward" as their inference route.